apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/Components/ModifyDatesModal/ModifyDatesModal.spec.ts

```diff
@@ -5,7 +5,7 @@ import { createCaseModifiedExplanation } from './ModifyDatesModal'
 describe('createCaseModifiedExplanation', () => {
   const formatMessage = createIntl({ locale: 'is', onError: jest.fn() })
     .formatMessage
-  beforeAll(() => jest.useFakeTimers('modern'))
+  beforeAll(() => jest.useFakeTimers())
```

Comment:
It seems like the new way of setting the time in jest tests is setting the `now` field in the `fakeTimersConfig` 
```suggestion
  beforeAll(() => jest.useFakeTimers({ now: new Date('2022-06-13T13:37:00Z') }))
```
https://jestjs.io/docs/jest-object#fake-timers

Reply:
Are we ok with using something arbitrary like '2022-06-13T13:37:00Z' here?

Reply:
yes, for this test it is fine.

Reply:
Seeing as these tests are succeeding, I think we should let it slip for now, this PR is not supposed to go into changing the functionality of tests.

My assumption was that the faketimers now use 'modern' by default, I will let this one be while I fixed the other one where a date was being used.